packages/auto-approve/src/process-checks/owl-bot-template-changes.ts

```diff
@@ -96,13 +96,13 @@ export class OwlBotTemplateChanges extends Process implements LanguageRule {
 
     reportIndividualChecks(
       ['authorshipMatches', 'titleMatches', 'bodyMatches', 'isGAPIC'],
-      [authorshipMatches, !titleMatches, !bodyMatches, isGAPIC],
+      [authorshipMatches, titleMatches, !bodyMatches, isGAPIC],
```

Comment:
Technically this is an unrelated change, just a little bug I noticed with logging. We were reporting the inverse of what titleMatches should show

Reply:
I suggest you separate it into 2 PRs.

Reply:
`reportIndividualChecks` only relates to logging, so it won't break anything to make this change. I would prefer to bundle it given that it would only be a single character change, if that's alright.

Reply:
I still think you should separate.

It's hard to revert just one part.

Reply:
I agree on this point too.